src/agent.cpp

```diff
@@ -762,7 +774,8 @@ Agent::doReorient()
 	dlog.addText( Logger::TEAM,
                       __FILE__": invalid my vel" );
       }
-      return Bhv_Emergency().execute( this ); // includes change view
+      Bhv_Emergency().execute( this ); // includes change view
```

Comment:
Emergency always returns true, so lets just return it: 
return Bhv_Emergency().execute( this );